tests/lib/rules/strict-component-boundaries.js

```diff
@@ -11,51 +11,62 @@ const parserOptions = {
 const errors = [
   {
     type: 'ImportDeclaration',
-    message: 'Strict component boundaries.',
+    message: `Do not reach into an individual component's folder for nested modules. Import from the closest shared components folder instead.`,
```

Comment:
cc @GoodForOneFare with regards to https://github.com/Shopify/eslint-plugin-shopify/issues/122